main.go

```diff
@@ -938,14 +943,44 @@ Vault will remain sealed).
 	})
 
 	r.Dispatch("env", &Help{
-		Summary: "Print the VAULT_ADDR and VAULT_TOKEN for the current target",
+		Summary: "Print the environment variables for the current target",
 		Usage:   "safe env",
-		Type:    AdministrativeCommand,
+		Description: `
+Print the environment variables representing the current target.
+
+ --format FORMAT   Format used to render the environment variables. Supported
+                   values are "simple" (default), "bash", and "fish".
+
+Please note that if you specify the output format to be either "bash" or
+"fish", then the output will be written to STDOUT instead of STDERR to make it
+easier to consume.
+		`,
+		Type: AdministrativeCommand,
 	}, func(command string, args ...string) error {
 		rc.Apply(opt.UseTarget)
-		fmt.Fprintf(os.Stderr, "  @B{VAULT_ADDR}  @G{%s}\n", os.Getenv("VAULT_ADDR"))
-		fmt.Fprintf(os.Stderr, "  @B{VAULT_TOKEN} @G{%s}\n", os.Getenv("VAULT_TOKEN"))
-		return nil
+		var print envPrintFunc
+		var out io.Writer
+
+		switch opt.Env.Format {
+		case "bash":
+			print = printEnvForBash
+			out = os.Stdout
+		case "fish":
+			print = printEnvForFish
+			out = os.Stdout
+		case "simple":
+			print = printEnv
+			out = os.Stderr
+		default:
+			return fmt.Errorf("Unrecognized format '%s'", opt.Env.Format)
+		}
+
+		vars := map[string]string{
+			"VAULT_ADDR":        os.Getenv("VAULT_ADDR"),
+			"VAULT_TOKEN":       os.Getenv("VAULT_TOKEN"),
+			"VAULT_SKIP_VERIFY": os.Getenv("VAULT_SKIP_VERIFY"),
+		}
+		return print(out, vars)
```

Comment:
Inline the `printEnvFor*` functions, and don't bother with the intermediate map.

Reply:
Will do :)